keplergl/kepler.gl
# Fixes a number of issues preventing Kepler from building on fresh checkout

When I perform a fresh clone, a number of build issues prevent Kepler from starting successfully. 

This collection of changes ensures that the application can be cloned / installed / built without error.

## Testing

1. Clone this PR into a fresh installation
2. Run `yarn bootstrap`
3. Run `yarn start`

Everything should build successfully and the demo app should load without error.

May be related to #2594, #2593, and earlier reports.
